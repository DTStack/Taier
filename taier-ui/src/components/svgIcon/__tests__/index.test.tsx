import { render } from '@testing-library/react';
import SvgIcon from '..';

describe('Test SvgIcon Component', () => {
    it('Should match snapshot', () => {
        expect(render(<SvgIcon linkHref="iconicon_copy" />).asFragment()).toMatchSnapshot();
    });
});
