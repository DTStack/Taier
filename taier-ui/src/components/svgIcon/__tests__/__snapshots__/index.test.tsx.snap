// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test SvgIcon Component Should match snapshot 1`] = `
<DocumentFragment>
  <svg
    aria-hidden="true"
    class="icon-svg default-icon "
    linkhref="iconicon_copy"
  >
    <use
      xlink:href="#iconicon_copy"
    />
  </svg>
</DocumentFragment>
`;
